src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -176,10 +190,13 @@ public String getDisplayName() {
             return Messages.AWSCredentialsImpl_DisplayName();
         }
 
+        public static final Integer defaultStsTokenDuration = STS_CREDENTIALS_DURATION_SECONDS;
```

Comment:
Do we need another constant here? If it is to reference it from Jelly, `${it.STS_CREDENTIALS_DURATION_SECONDS}` should do the trick. Anyway by convention, constants should be all uppercase.

Reply:
I kept the original constant for backwards compatibility, as it was public. I tried incorporating your suggestion, but could not access `STS_CREDENTIALS_DURATION_SECONDS` from Jelly using the `it` syntax, so at the moment, I've kept it as it was, but changed the casing. Please let me know if this would be okay.